src/sentry/static/sentry/app/components/events/interfaces/openInButton.jsx

```diff
@@ -0,0 +1,99 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import SentryTypes from 'app/sentryTypes';
+import Button from 'app/components/button';
+import qs from 'query-string';
+import styled from 'react-emotion';
+import {t} from 'app/locale';
+
+import withApi from 'app/utils/withApi';
+import withLatestContext from 'app/utils/withLatestContext';
+
+class OpenInButton extends React.Component {
+  static propTypes = {
+    api: PropTypes.object,
+    organization: SentryTypes.Organization,
+    lineNo: PropTypes.number,
+    filename: PropTypes.string,
+    group: SentryTypes.Group,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: false,
+      error: false,
+      components: [],
+    };
+  }
+
+  componentWillMount() {
+    this.fetchIssueLinkComponents();
+  }
+
+  fetchIssueLinkComponents() {
+    const {api, organization, group} = this.props;
+    if (group && group.project) {
+      api
+        .requestPromise(
+          `/organizations/${organization.slug}/sentry-app-components/?filter=stacktrace-link&projectId=${group
+            .project.id}`
+        )
+        .then(data => {
+          if (data.length) {
+            this.setState({components: data});
+          }
+        })
+        .catch(error => {
+          return;
+        });
```

Comment:
We should put a feature check for `sentry-apps`  around this whole thing so that we don't make all these requests if there's no way they're going to return anything.